pkgs/sdk-pkg/src/genie/libs/sdk/apis/iosxe/eigrp/get.py

```diff
@@ -0,0 +1,210 @@
+# =================================================
+# Get APIs for 'EIGRPv4 and EIGRPv6' command
+#Author: Mehdi Cherifi
+#LinkedIn: https://www.linkedin.com/in/mehdi-cherifi
+#Twitter: https://twitter.com/LocketKeepsake
+#Github: https://github.com/cherifimehdi
+# =================================================
+
+import logging
+import os
+from genie.metaparser.util.exceptions import SchemaEmptyParserError
+log = logging.getLogger(__name__)
+
+
+def get_eigrp_interfaces(device, vrf='default', AS_N=None, ip='ipv4'):
+   """Gets all the interfaces on the device for a given AS and vrf for ipv4 or ipv6 from -show ip eigrp interfaces- for ipv4 by default or -show ipv6 eigrp interfaces- for ipv6
+
+      Args:
+        device (obj): Device object
+        vrf = "default" (str): Name of the vrf by default set to "default"
+        AS_N = None (int): Autonomous System
```

Comment:
please provide a better name for variable

Reply:
For the variable I think I used the same recommendation and it is significant (for example AS_N: Autonomous System Number)